### DISCUSSION TITLE
why not show all posts in hexo.locals.get('posts') ?

### DISCUSSION THREAD

**Discussion by jerryc127** [hexojs/hexo#5230] [Question / Answer]
I create the tags plugin

```
function series () {
   console.log(hexo.locals.get('posts').length)
}

hexo.extend.tag.register('series', series, { ends: false })
```

it show 2, but i have 26 posts ?

and the another question

i want to store the data, and then i can read it in hexo.extend.helper.register and  hexo.extend.tag.register.
i try the fragment_cache method to store data. but  it seem like  hexo.extend.tag.register cannot use  fragment_cache .

**Comment by D-Sketon** [comment]
I think this may be a bug because I could reproduce the situation and I found that the remaining posts were in the _mutex _queue.
![image](https://github.com/hexojs/hexo/assets/49871906/f1d81ecf-0c06-439f-921e-c9d60cbaf9a8)
